packages/react-components/react-persona/src/components/Persona/usePersonaStyles.ts

```diff
@@ -15,33 +15,35 @@ export const personaClassNames: SlotClassNames<PersonaSlots> = {
 
 const avatarSpacing = `--fui-Persona__avatar--spacing`;
 
+const useRootStyles = makeResetStyles({
```

Comment:
```suggestion
const useRootClassName = makeResetStyles({
```

Reply:
Hmmm...I think something got mixed up because this change should not be in this PR. Will back these changes out of this one.